features/leeftijd_bepaling.feature

```diff
@@ -35,11 +35,27 @@ Scenario: Alleen jaar van geboorte datum is bekend
 	Dan is attribuut leeftijd niet aanwezig
 
 Scenario: Persoon is overleden
-	Gegeven een ingeschreven persoon met geboortedatum 26 mei 1953
-	En de ingeschreven persoon is overleden
-	Als de ingeschreven persoon op <raadpleeg datum> wordt geraadpleegd
+	Gegeven de persoon met burgerservicenummer "999993197" is overleden op 28-04-2019
+	Als de ingeschreven persoon met burgerservicenummer 999994268 wordt geraadpleegd met fields=leeftijd
```

Comment:
In de Gegeven staat een bsn 999993197 en in de Als staat een bsn 999994268. Klopt dit?

Reply:
nee klopt niet. heb het gecorrigeerd